test/Broadway/CommandHandling/EventDispatchingCommandBusTest.php

```diff
@@ -13,29 +13,41 @@
 
 namespace Broadway\CommandHandling;
 
+use Broadway\EventDispatcher\EventDispatcher;
 use Broadway\TestCase;
 
 class EventDispatchingCommandBusTest extends TestCase
 {
+    /**
+     * @var CommandBus|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
I'm not a big fan of these annotations because they are not automatically validated.
In case of mocking I think it's best to just specify the base class not the mock.

Reply:
Without this annotation the code completion of an IDE is not working. That's why I added it. But I can remove it.

Reply:
ok, that's a fair reason :)
you can leave it.